packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -58,6 +62,9 @@ interface ExpandedTutorialStep {
 
 type CheckStatus = 'INDETERMINATE' | 'CHECKING' | 'PASSED' | 'FAILED';
 
+let hasRecordedStart = false;
```

Comment:
This is a module scoped variable, so does it assume new tutorials are always loaded with a full page reload? Seems like that could be a little brittle, since a refactoring to make this a little more SPAish could break that assumption.